linked_list/linked_list.py

```diff
@@ -23,7 +23,14 @@ def __init__(self):
     self.tail = None
 
   def add_to_tail(self, value):
-    pass
+
+    node = Node(value)
+
+    if self.tail is not None:
+      self.tail.set_next(node)
+    else:
+      self.head = node
+    self.tail = node
 
   def remove_head(self):
     pass
```

Comment:
Keep working on remove_head, contains, and get_max to see if you can pass the tests.